### DISCUSSION TITLE
Bookmark details are fetched in different language

### DISCUSSION THREAD

**Discussion by herndlm** [nextcloud/bookmarks#2237] [Q&A]
Hi, I'm German speaking but use English on my computer/server and also in Nextcloud. But when I added e.g. https://www.imdb.com/title/tt0084787/ as bookmark it fetched the details in German instead of English. I assume this must be coming from _somewhere_ I'm not thinking of.

Any ideas where to configure or change this?

**Comment by marcelklehr** [comment]
Mh, that's strange. The User Agent used is `Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:87.0) Gecko/20100101 Firefox/87.0` maybe that gives a clue
Is your Nextcloud hosted in Germany, perhaps?

**Reply by herndlm** [reply]
Basically yes, but I'm self-hosting it on my raspberry pi where I could control server settings and such.
